### PR TITLE
Added check for entryPointAddress field

### PR DESCRIPTION
**Pull request by VGabriel45**
# Summary

Added conditional logic for situations when user will not provide "entryPointAddress" to the Bundler.

Related Issue: # 

## Change Type

- [x] Bug Fix
- [ ] Refactor
- [ ] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

# Checklist

- [x] My code follows this project's style guidelines
- [x] I've reviewed my own code
- [ ] I've added comments for any hard-to-understand areas
- [ ] I've updated the documentation if necessary
- [x] My changes generate no new warnings
- [ ] I've added tests that prove my fix is effective or my feature works
- [x] All unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: packages/bundler/src/utils/Constants.ts [hunk 1]
```diff
@@ -125,3 +125,5 @@ export const UserOpWaitForTxHashMaxDurationIntervals: { [key in ChainId]?: numbe
   [ChainId.CHILIZ_MAINNET]: 20000,
   [ChainId.CHILIZ_TESTNET]: 20000,
 };
+
+export const DEFAULT_ENTRYPOINT_ADDRESS = "0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789";
```
Comment (AmanRaj1608) [inline packages/bundler/src/utils/Constants.ts:129]: could also import this from the baseValidationModule, but this is also fine if we don't wanna have circular dependency

**Review by AmanRaj1608 — CHANGES_REQUESTED** [review]
Fix lint issues

File: packages/bundler/src/utils/Constants.ts (hunk 1 above)
Comment (VGabriel45) [reply packages/bundler/src/utils/Constants.ts:129]: If it's fine like this I'm gonna let it

**Review by livingrockrises — APPROVED** [review]
LGTM

**Review by AmanRaj1608 — APPROVED** [review]

**Pull request merged**